tests/unit/async_/test_bookmark_manager.py

```diff
@@ -0,0 +1,295 @@
+# Copyright (c) "Neo4j"
+# Neo4j Sweden AB [https://neo4j.com]
+#
+# This file is part of Neo4j.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import annotations
+
+import itertools
+import typing as t
+
+import pytest
+
+import neo4j
+from neo4j._async.bookmark_manager import AsyncNeo4jBookmarkManager
+from neo4j._async_compat.util import AsyncUtil
+from neo4j.api import Bookmarks
+
+from ..._async_compat import mark_async_test
+
+
+@pytest.mark.parametrize("db", ("foobar", "system"))
+@mark_async_test
+async def test_return_empty_if_db_doesnt_exists(db) -> None:
+    bmm = neo4j.AsyncGraphDatabase.bookmark_manager()
+
+    assert set(await bmm.get_bookmarks(db)) == set()
+
+
+@pytest.mark.parametrize("db", ("db1", "db2", "db3"))
```

Comment:
magic

Reply:
pytest's parametrization feature is incredible mighty :clap: